### ISSUE TITLE
Some classes cannot be created in angular

### ISSUE THREAD

**Issue by macsux** [surveyjs/survey-library#1937]
### Are you requesting a feature, reporting a bug or asking a question?
Bug (probably)

### What is the current behavior?
I'm following example here: https://surveyjs.answerdesk.io/ticket/details/t2587/customize-imagepicker-template
to change template of some of the rendered questions. 
When using surveyjs-angular, I'm unable to update template
`new Survey.SurveyTemplateText().replaceText(template, "question", "text");`

I'm getting `Error: Survey.SurveyTemplateText is not a constructor`

I've noticed a lot of other classes declared in `survey.angular.d.ts` cannot be instantiated with the same error

### What is the expected behavior?
I'm able to execute the line

### How would you reproduce the current behavior (if this is a bug)?
https://stackblitz.com/edit/surveyjs-angular5-example-un7d7b?file=app%2Fsurveyjs.component.ts

### Specify your

* browser: chrome
* browser version: 79
* surveyjs platform (angular or react or jquery or knockout or vue): angular
* surveyjs version: 1.1.24

**Comment by tsv2013** [comment]
As you can see in this thread - https://surveyjs.answerdesk.io/ticket/details/t2587/customize-imagepicker-template
Actually direct template customization approach is not recommended.
At his moment these classes are not exported in survey-angular module.
This is a bug of .d.ts bundling.

It is better to use the https://surveyjs.io/Documentation/Library/?id=surveymodel#onAfterRenderQuestion event as shown in this sample - https://surveyjs.io/Examples/Library/?id=survey-afterrender&platform=jQuery&theme=modern

**Comment by macsux** [comment]
I saw that, but team feels direct dom post render manipulation feels much
dirtier and significantly breaks away from angular paradigm. If this is a
big, is this something that can be added to typescript exports?

**Comment by tsv2013** [comment]
I see your point. We'll try to return these classes to exports. I'll update this thread in a couple days. If not please feel free to ping me here.

**Comment by lovemyway** [comment]
Hey, any updates on this?

**Comment by tsv2013** [comment]
@macsux @lovemyway The thing is that e.g. mentioned `SurveyTemplateText` class is from the `survey-knockout` library, as well as some other classes recently were available in the `survey-angular` library.

As you probably know the `survey-angular` is a wrapper over the `survey-knockout` library. And `survey-angular` indeed contains KnockoutJS + `survey-knockout` + survey-angular wrapper code.
If classes from `survey-knockout` are exposed from the `survey-angular` library typings it looks like `survey-angular` depends on KnockoutJS (that is actually wrong) but some classes from `survey-knockout` are exposed KnockoutJS specific type properties - `ko.observable` and so on.

Some `survey-angular` users complained about this. And we decided to get rid of Knockout typings in the `survey-angular.d.ts` file. This lead to excluding `survey-knockout` classes like  `SurveyTemplateText` were removed from the `.d.ts` file. But `survey-angular.d.ts` file unfortunately still contains reference to the KnockoutJS typings (https://surveyjs.answerdesk.io/ticket/details/t3116/trying-to-determine-which-version-of-survey-angular-and-associated-libraries-to-use-with).

We're still looking for the best solution for the issue and will update this thread as soon as it will fe found,
